benchmark/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.10)
 project(benchmark)
 
-set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD 23)
```

Comment:
@TeachRaccooon, this is actually the second place where you specify the C++ standard. So you've fixed it in ``CMake/rl_build_options.cmake``, but you haven't fixed it here yet. Please change this line to C++ 20.

Reply:
Done.